src/mono/mono/mini/mini.h

```diff
@@ -1827,7 +1827,9 @@ enum {
 #define OP_PCLT OP_LCLT
 #define OP_PCGT OP_LCGT
 #define OP_PCLT_UN OP_LCLT_UN
+#define OP_PCLE_UN OP_LCLE_UN
 #define OP_PCGT_UN OP_LCGT_UN
+#define OP_PCGE_UN OP_LCGE_UN
```

Comment:
@vargaz, it looks like while OP_ICLE_UN and OP_ICGE_UN exist, OP_LCLE_UN and OP_LCGE_UN don't. What would you recommend at the right way to implement this? Remove these defines and update intrinsics.c with an implementation that emits eg a cgt and ceq for less than or equal?